.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -343,17 +348,17 @@ kernel_matrix_testing_setup_env:
     - !reference [.kernel_matrix_testing_new_profile]
     - !reference [.write_ssh_key_file]
   script:
-    - ALL_INSTANCE_IPS=$(cat $CI_PROJECT_DIR/stack.outputs | grep -e '-instance-ip' | cut -d ' ' -f 2 | tr '\n' ' ' | awk '{$1=$1};1' | tr ' ' ',')
+    - INSTANCE_IP=$(cat $CI_PROJECT_DIR/stack.output | grep -e '-instance-ip' | cut -d ' ' -f 2 | tr '\n' ' ' | awk '{$1=$1};1' | tr ' ' ',')
```

Comment:
this should be `INSTANCE_IP=$(grep "$ARCH-instance-ip" $CI_PROJECT_DIR/stack.output | cut -d ' ' -f 2 | tr -d '\n')` and the later instance can be removed right? We shouldn't be checking for both instances here.

Reply:
Ah yes!